src/app/retrospective/retrospective-list/retrospective-list.component.ts

```diff
@@ -1,26 +1,34 @@
 import { ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
-import { RetrospectiveListDataSource } from './retrospective-list.data-source';
```

Comment:
Shouldn't we remove `RetrospectiveListDataSource`, if it is not being used?

Reply:
It's already removed.